pkg/template/api/v1/types.go

```diff
@@ -14,15 +14,23 @@ type Template struct {
 	// Standard object's metadata.
 	kapi.ObjectMeta `json:"metadata,omitempty"`
 
-	// Objects is an array of objects to include in this template. Required.
+	// message is an optional instructional message that will
+	// be displayed when this template is instantiated.
```

Comment:
> would you be happier with "optional message that may be displayed depending which client you use and whether it bothers to support this field"?

Yeah, along with, "in our default client, we ignore this in the default processing/creation workflow".